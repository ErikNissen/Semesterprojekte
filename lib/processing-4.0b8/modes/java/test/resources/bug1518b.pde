version https://git-lfs.github.com/spec/v1
oid sha256:fcf9dbe0759fd2d7ede514f956dc6bfe1264167885803440a26c2193a88ac63d
size 258
