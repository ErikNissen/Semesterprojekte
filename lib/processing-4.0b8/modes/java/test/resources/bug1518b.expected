version https://git-lfs.github.com/spec/v1
oid sha256:87ebd6163f6e4482c27943083a7b33407f8e8c68efaece5c0df4197f53f9fdea
size 921
